plugins/vpp/wireguardplugin/vppcalls/vpp2009/wireguard_vppcalls.go

```diff
@@ -0,0 +1,87 @@
+//  Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vpp2009
+
+import (
+	"encoding/base64"
+	"fmt"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2009/interface_types"
+	vpp_wg "go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2009/wireguard"
+	wg "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/wireguard"
+)
+
+func (h *WgVppHandler) AddPeer(peer *wg.Peer) (uint32, error) {
+	inv_peer_idx := ^uint32(0)
+
+	peer_vpp := vpp_wg.WireguardPeer{
+		Port:                uint16(peer.Port),
+		PersistentKeepalive: uint16(peer.PersistentKeepalive),
+		TableID:             peer.TableId,
+		Flags:               vpp_wg.WireguardPeerFlags(peer.Flags),
+		NAllowedIps:         uint8(peer.NAllowedIp),
```

Comment:
There is no need to set value of `NAllowerIps`. The value is ignored and set to the slice length of `AllowedIps` field:

https://github.com/glazychev-art/vpp-agent/blob/wireguard_plugin/plugins/vpp/binapi/vpp2009/wireguard/wireguard.ba.go#L394